ui/radar/monsters.js

```diff
@@ -0,0 +1,787 @@
+'use strict';
+
+let Monsters = {
```

Comment:
Where did this list come from? It looks like the NotoriousMonster table in SaintCoinach (although xivapi doesn't like using this table) has all of this information in it, both names and rank.

Would it be possible to write a script like the weather or fishing script to automatically generate some `resources/hunts.js` file so I don't have to manually keep this up to date myself?

Reply:
This resource will only be updated when New Expansion releases (I think), so it's won't be frequently updated maybe?
But sure, an auto-update script is much more convenient, I'll look into this later.

Reply:
I think mostly my concern here is that I am happy to put extremely useful modules like this into cactbot proper that a lot of people would find useful and keep it up to date as things change.  However, everything that gets added is also extra friction for me (see also not enough time to fix all the fishing module issues), and so it's really important for me that anything that gets added can be kept up to date as automatically as possible with little effort as new content is added.

An auto-update script would also remove the need to manually translate this into french and german as well.

Reply:
You can use `util/get_radar_data.py` to auto-update now ~